### PR TITLE
MockDrivetrain

### PR DESCRIPTION
**Pull request by lachlanmcglennon**
added mock drivetrain and tests for all methods

### CODE REVIEW

File: code/src/test/java/frc/robot/mocks/subsystems/MockDrivetrain.java [hunk 1]
```diff
@@ -136,7 +137,7 @@ public void moveEast(double maxSpeed, double distance) {
     }
 
     @Override
-    public void moveWest(double distance) {
+    public void moveWest(double distance) {//do
         this.faceWest();
```
Comment (noam987) [inline code/src/test/java/frc/robot/mocks/subsystems/MockDrivetrain.java]: We should discuss if the moveDirection methods rotate the robot. I was under the impression that these methods were to move the robot in a field relative view. I.E. this method would move the robot to driver west, not caring about which direction the robot is facing.

**Review by noam987 — COMMENTED** [review]
why should the tank drive be in robot subsystems?  I don't understand the usefulness of having it in that folder, especially when it is not fleshed out. 

My reasoning is that I can't think of a positive yet, and it just increases clutter in that folder. 
Interested to hear other opinions.

**Comment by dotnetprofessional** [comment]
> why should the tank drive be in robot subsystems? I don't understand the usefulness of having it in that folder, especially when it is not fleshed out.
> 
> My reasoning is that I can't think of a positive yet, and it just increases clutter in that folder.
> Interested to hear other opinions.

This class can be deleted, it was a originally created as a sample of another drive train. However, it wasn't supposed to be checked in.

**Comment by noam987** [comment]
I'll delete that and I think It is good to merge

**Review by noam987 — APPROVED** [review]

**Comment by dotnetprofessional** [comment]
#7 Adding of unit test support

**Pull request merged**